USAGE.md

```diff
@@ -27,14 +27,23 @@ Killing the running process (ctrl+C) will stop the app. You can get rid of the D
 1. Create a .env file in the root directory of the project with the following, replacing (?) with the desired information: <br>
 ```
 OED_SERVER_PORT=?              // The port that the server should run on. 3000 is a good default choice
+OED_TOKEN_SECRET=?             // Token for authentication. Generate something secure and random
```

Comment:
I know this isn't part of your pull request, but can you change the comment types of all the env variables from `//` to `#`?